server/sonar-server/src/main/java/org/sonar/server/issue/notification/IssueChangeNotification.java

```diff
@@ -54,15 +54,19 @@ public IssueChangeNotification setIssue(DefaultIssue issue) {
   }
 
   public IssueChangeNotification setProject(ComponentDto project) {
-    return setProject(project.getKey(), project.name(), project.getBranch());
+    // TODO set the pull request instead of the branch
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fbranch%2Ffirst-class-pull-request&issues=AWGuRf5waHWQ5zPschp3&open=AWGuRf5waHWQ5zPschp3)

[ISSUE_KEY]: AWGuRf5waHWQ5zPschp3